app/processImage.js

```diff
@@ -48,26 +47,68 @@ function getCropFunction(cropType) {
 
 
 function applySmartCrop(image, cropSize, callback) {
-    smartcrop.crop(image, cropSize).then(function(result) {
-        var crop = result.topCrop;
-        sharp(image)
-            .extract({ width: crop.width, height: crop.height, left: crop.x, top: crop.y })
-            .resize(cropSize.width, cropSize.height)
-            .toBuffer(callback)
-    }, callback);
-}
+  const img = sharp(image);
```

Comment:
lets follow same indentation for the whole project. We are using 4 spaces. I'm Adding .editorconfig in few minutes.

Reply:
Merged with https://github.com/PrimeDigitalGlobal/image-resize-lambda/pull/13